packages/core/contracts/financial-templates/perpetual-multiparty/PositionManager.sol

```diff
@@ -141,8 +131,7 @@ contract PositionManager is FeePayer, AdministrateeInterface {
     }
 
     /**
-     * @notice Construct the PricelessPositionManager
-     * @param _expirationTimestamp unix timestamp of when the contract will expire.
+     * @notice Construct the PositionManager.
      * @param _withdrawalLiveness liveness delay, in seconds, for pending withdrawals.
      * @param _collateralAddress ERC20 token used as collateral for all positions.
      * @param _finderAddress UMA protocol Finder used to discover other protocol contracts.
```

Comment:
FYI the latest plan is to have two finder addresses, one pointing to DVM contracts and one to financial contract addresses like the funding rate store & token factory. We should probably push to separate PR but just wanted to highlight this archictecture change

Reply:
Yup. Also your original Issue #2016 has these in the interface but I thought it best to leave it out for now, just like with the funding rate stuff.